code/modules/mob/living/simple_animal/hostile/terror_spiders/white.dm

```diff
@@ -40,7 +40,8 @@
 	var/inject_target = pick("chest","head")
 	L.attack_animal(src)
 	L.KnockDown(10 SECONDS)
-	if(L.IsStunned() || L.IsParalyzed() || L.can_inject(null, FALSE, inject_target, FALSE))
+	L.adjustStaminaLoss(30)
+	if(L.can_inject(null, FALSE, inject_target, FALSE) || (HAS_TRAIT(L, TRAIT_HANDS_BLOCKED) && HAS_TRAIT(L, TRAIT_IMMOBILIZED)))
```

Comment:
ditto here

Reply:
i did that there

Reply:
im blind

Reply:
i'm blind too cuz i managed to only do it once in the code